server/src/main/java/org/elasticsearch/index/mapper/FieldMapper.java

```diff
@@ -250,7 +260,7 @@ public final void executeScript(
             if ("continue".equals(onScriptError)) {
                 documentParserContext.addIgnoredField(name());
             } else {
-                throw new MapperParsingException("Error executing script on field [" + name() + "]", e);
+                throw new DocumentParsingException(XContentLocation.UNKNOWN, "Error executing script on field [" + name() + "]", e);
```

Comment:
this makes me wonder if we should use this exception when we don't have a location. Is this a document parsing error?

Reply:
I think this still counts as an error in document parsing?  It's still definitely not a mapper parsing exception...